scripts/hail_batch/hgdp1kg_tobwgs_pca_nfe_new_variants_no_outliers/hgdp_1kg_tob_wgs_nfe_pca_no_outliers.py

```diff
@@ -38,15 +38,8 @@ def query():
         & (mt.s != 'TOB1668')
     )
 
-    # Remove related samples (at the 2nd degree or closer)
-    pc_rel = hl.pc_relate(mt.GT, 0.01, k=20, statistics='kin')
-    pairs = pc_rel.filter(pc_rel['kin'] >= 0.125)
-    related_samples_to_remove = hl.maximal_independent_set(pairs.i, pairs.j, False)
-    n_related_samples = related_samples_to_remove.count()
-    print(f'related_samples_to_remove.count() = {n_related_samples}')
-    mt = mt.filter_cols(
-        hl.is_defined(related_samples_to_remove[mt.col_key]), keep=False
-    )
+    # Remove related samples at the 2nd degree or closer, as indicated by gnomAD
+    mt = mt.filter_cols(mt.hgdp_1kg_metadata.gnomad_release is True)
```

Comment:
1. Won't this also filter out the `TOB` samples?
1. Do you need the `is True` or does `mt = mt.filter_cols(mt.hgdp_1kg_metadata.gnomad_release)` work as well?

Reply:
good point, I don't need the 'is True' at the end. I've also add in `mt.s.contains('TOB')` so TOB samples aren't filtered out.